network/some_ip/some_ip.ksy

```diff
@@ -0,0 +1,159 @@
+meta:
+  id: some_ip
+  title: AUTOSAR SOME/IP
+  license: CC0-1.0
+  ks-version: 0.9
```

Comment:
I wonder why you declare `0.9` as the lowest possible compiler version? I don't think that I've seen some 0.9 feature (well, I've not tested it), so I hope it would properly compile even with the KSC 0.8.

But rather don't change it without testing that it really works under version 0.8, I'm just curious if you had encountered some issue with the 0.8 version or you needed some 0.9 feature.

Reply:
I just used KSC v0.9 only, since the python code compiled with v0.8 didn't work. There was an issue with circular references which had been fixed with v0.9.